ReSharper.FSharp/test/data/cache/csharpResolve/tooltips/XmlDoc 13 - Type members 02 - Abstract member.fsi

```diff
@@ -0,0 +1,8 @@
+module Test
+
+[<AbstractClass>]
+type A =
+    /// M signature
+    abstract member M: unit -> unit
+    /// M signature default
```

Comment:
It's a very interesting case, nice find. We should probably emit some warning here?

Reply:
yes, sure